Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -300,8 +301,31 @@ its :php:`__invoke()` method will be called:
 
 Read :ref:`how to configure dependency injection in extensions <dependency-injection-in-extensions>`.
 
+Existing event listeners can be overridden by custom implementations. For example,
+:file:`EXT:news/Configuration/Services.yaml` could listen on the event
+:php:`TYPO3\CMS\Frontend\Event\ModifyHrefLangTagsEvent`:
 
+..  literalinclude:: _ServicesOverrideBase.yaml
+    :language: yaml
+    :caption: EXT:my_extension/Configuration/Services.yaml
+
+If you want to replace this event listener with your custom implementation, your extension can
+achieve this by specifying:
+
+..  literalinclude:: _ServicesOverrideBase.yaml
+    :language: yaml
+    :caption: EXT:my_extension/Configuration/Services.yaml
 
+Make sure that you set the :yaml:`identifier` attribute to exactly the string that the
+original implementation uses. If that attribute is not mentioned specifically in the original
+implementation, the fully qualified name of the :php:`EventListener` class is used. So if
+:yaml:`identifier: 'ext-news/modify-hreflang'` would not be there, the identifier would be set
+to :yaml:`identifier: 'GeorgRinger\News\Seo\HrefLangEvent'` and you would need to use that
+identifier in your implementation.
+
+Note that overriding listeners requires your extension to declare a dependency on the :php:`EXT:news`
+extension (through :file:`composer.json`). This ensures the loading order allows your extension
+to be evaluated after the extension you want to override.
```

Comment:
I'd suggest to add a additional mentioning that the same encounts for a `ext_emconf.php` for non-composer installations and upload to TER.

Reply:
Good note, I always forget that ext_emconf still has some usage.